README.md

```diff
@@ -1,272 +1,27 @@
 # Language Forge #
 
-[Language Forge](https://languageforge.org) is an online service to collaborate on building a dictionary.
+[Language Forge](https://languageforge.org) is an online service to collaborate on building a dictionary.  Users should head on over to [languageforge.org](https://languageforge.org)
 
-## Users ##
+### Reporting Problems with the Language Forge Application ###
 
-To use **Language Forge** go to [languageforge.org](https://languageforge.org).
+If you are reading this, then there is a chance that you have a GitHub account and can create an issue directly in our [issue tracker](https://github.com/sillsdev/web-languageforge/issues). (Please note that all information submitted to our [issue tracker](https://github.com/sillsdev/web-languageforge/issues) is public information).
 
-### User Problems ###
+If you don't have a GitHub account or don't want your issue to be public, you can send an email to "languageforgeissues @ sil dot org".
 
-To report an issue using the **Language Forge** service, email "languageforgeissues @ sil dot org".
+## Other Resources ##
 
-## Special Thanks To ##
-- ![BrowserStack Logo](readme_images/browserstack-logo.png "BrowserStack") for mobile device testing.
-- [![Bugsnag logo](readme_images/bugsnag-logo.png "Bugsnag")](https://bugsnag.com/blog/bugsnag-loves-open-source) for error reporting.
+[Acknowledgements](docs/ACKNOWLEDGEMENTS.md)
```

Comment:
These might look better on separate lines.

Reply:
I think an explicit `<br/>` at the end of each line might be needed. I believe Markdown uses trailing spaces before the newline character to indicate "Really break the line here", but that's a problem for most VS Code setups that trim trailing spaces.

Or, of course, turn this into an unordered list like:

```md
* Acknowledgements
* Known Issues
* etc.
```

Reply:
I turned it into an unordered list

Reply:
I think @rmunn is fine with me resolving this.